README.md

```diff
@@ -13,6 +13,8 @@ mainboards:
 - [MSI Z790-P PRO](./boards/msi/ms7e06) -- project [Raptor Lake Desktop](https://nlnet.nl/project/RaptorLake/)
 - [NovaCustom V54](./boards/NovaCustom/V54x_MTL/)
 - [NovaCustom V56](./boards/NovaCustom/V56x_MTL/)
+- [Protectli VP66xx](./boards/Protectli/VP66xx/)
```

Comment:
The results for v66xx have already been published and should not be included here.